lstmcpipe/__init__.py

```diff
@@ -1,4 +1,4 @@
 from pathlib import Path
 
 lstmcpipe_root_dir = Path(__file__).absolute().parents[1]
-__version__ = "0.6.1"
```

Comment:
you should still import `__version__` here so one can do `from lstmcpipe import __version__`